dev-client/src/components/tables/soilProperties/SoilPropertiesDataTable.tsx

```diff
@@ -92,6 +92,15 @@ export const SoilPropertiesDataTable = ({rows}: Props) => {
     );
   };
 
+  const emptyRow = (
+    <Row justifyContent="flex-start">
+      <DataTableCell text="" width={columnWidthDepth} />
+      <DataTableCell text="" width={columnWidthTexture} />
+      <DataTableCell text="" width={columnWidthRockFragment} />
+      <DataTableCell text="" width={columnWidthColor} />
```

Comment:
question: is `text=""` needed?

Reply:
question: why do we want an empty row?

Reply:
1. `DataTableCell` expects a required `text` property, which is a `string` type. I could go either way on making `text` optional and having DataTableCell handle the case where it's not supplied, but I feel like it's just as reasonable as-is.

2. We only want it in the case where there are no depth intervals specified (like if the site has "Custom " soil pit depths but then no depths were added, or if the site is part of a project that has "None specified" for soil pit depth intervals)